### DISCUSSION TITLE
how to deploy on k3s

### DISCUSSION THREAD

**Discussion by xinmans** [argoproj/argo-rollouts#3052] [General]
xinmans@k3smaster-1:~$ curl -LO https://github.com/argoproj/argo-rollouts/releases/latest/download/kubectl-argo-rollouts-darwin-amd64
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 95.4M  100 95.4M    0     0  7685k      0  0:00:12  0:00:12 --:--:-- 10.4M
xinmans@k3smaster-1:~$ chmod +x ./kubectl-argo-rollouts-darwin-amd64
xinmans@k3smaster-1:~$ sudo mv ./kubectl-argo-rollouts-darwin-amd64 /usr/local/bin/kubectl-argo-rollouts
xinmans@k3smaster-1:~$ kubectl argo rollouts version
Error: exec format error

**Comment by kostis-codefresh** [comment]
The exec format error implies that your architecture is wrong. Where is this terminal running? If it is on x86 you cannot use the Mac binary there.
